recipes/python-systemd/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "python-systemd" %}
+{% set version = "235" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/systemd-python/systemd-python-{{ version }}.tar.gz
+  sha256: 4e57f39797fd5d9e2d22b8806a252d7c0106c936039d1e71c8c6b8008e695c0a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - pkg-config
+    - {{ cdt('systemd-devel') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - systemd
```

[`__init__` is blank](https://github.com/systemd/python-systemd/blob/main/systemd/__init__.py). Can we import other modules?

```suggestion
    - systemd
    - systemd.journal
    - systemd.daemon
```